script/dev-deploy

```diff
@@ -351,9 +351,11 @@ class ZookeeperService(Service):
         metrics_provider=False,
         **kargs,
     ):
-
-        docker_opts = "--rm " if rm else ""
-        docker_opts += self.zk_ex_container_opts + " "
+        docker_opts = []
+        if rm:
+            docker_opts.append("--rm")
+        docker_opts.append(self.zk_ex_container_opts + " ")
+        docker_opts = " ".join(docker_opts)
```

Comment:
Well, IMO,  there is no difference between this way of writing and the previous direct bare spelling of the string. I think the way I mentioned before is better for the following reasons:
1. after abstracting these functions into global helper functions, you can use these functions directly in all services
2. by naming the helper functions, the code is more readable. Each time you call a helper function that adds a configuration, you can directly know what configuration is added by the function name
3. Don't be afraid to splice strings directly in python, the compiler is optimized for this scenario without much performance loss.